cloudant/python-cloudant
# Fix Document.get_attachment bug

## What

Fix the Document `get_attachment` method so that it now correctly creates text and binary files as exepected as well as returns text, binary, and json content appropriately.

## How

- Make the `attachment_type` method argument optional.
- Add logic to figure out what type of content should be returned by the method as well as what kind of file should be created if a `write_to` argument is provided.

## Testing

- Add a test to verify getting a text attachment and writing it to a file.
- Add a test to verify getting a json attachment and writing it to a file.
- Add a test to verify getting a binary attachment and writing it to a file.

## Reviewers

reviewer: @emlaver 
reviewer: @ricellis 

## Issues

- #102
